config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/EncassEntityBuilder.java

```diff
@@ -80,7 +73,7 @@ public Integer getOrder() {
         return ORDER;
     }
 
-    private String getPolicyId(String policyWithPath, Bundle bundle, AnnotatedBundle annotatedBundle) {
+    private String getPolicyId(String policyWithPath, Bundle bundle) {
```

Comment:
dependencies should be taken from main bundle file, if we are passing AnnotatedBundle then dependencies should be taken from main bundle present in the AnnotatedBundle